src/address.js

```diff
@@ -32,13 +32,24 @@ function fromBech32 (address) {
 }
 
 function toBase58Check (hash, version) {
```

Comment:
And here,  again,  `version` would need to be replaced with `network`

Reply:
https://github.com/bitcoinjs/bitcoinjs-lib/blob/080675a5cc0a3be4c18a2d07be48f00bbf981d1e/src/address.js#L54 and https://github.com/bitcoinjs/bitcoinjs-lib/blob/080675a5cc0a3be4c18a2d07be48f00bbf981d1e/src/address.js#L55 use output script context to determine the version as network.pubKeyHash or network.scriptHash.  If the network is passed as a parameter to `toBase58Check(hash, network)` instead of version then it will also need that context to determine which is the version to use.  Seems like if all toBase58Check() needs is a version then maybe it should continue to take a version instead of the network?

Reply:
I added the network parameter to `toBase58Check()`.